src/Apps/Order2/Routes/Checkout/CheckoutContext/Order2CheckoutContext.tsx

```diff
@@ -553,131 +549,11 @@ export const Order2CheckoutContextProvider: React.FC<
 
   return (
     <Order2CheckoutContext.Provider runtimeModel={runtimeModel}>
-      <CheckoutLoadingManager orderData={orderData}>
-        {children}
-      </CheckoutLoadingManager>
+      {children}
     </Order2CheckoutContext.Provider>
   )
 }
 
-const CheckoutLoadingManager: React.FC<{
-  orderData: Order2CheckoutContext_order$data
-  children: React.ReactNode
-}> = ({ orderData, children }) => {
-  const [minimumLoadingPassed, setMinimumLoadingPassed] = useState(false)
-  const [orderValidated, setOrderValidated] = useState(false)
-  const [isStripeRedirectHandled, setIsStripeRedirectHandled] = useState(false)
-
-  // Handle Stripe redirect and call onComplete when done
-  useStripePaymentBySetupIntentId(orderData.internalID, () => {
-    setIsStripeRedirectHandled(true)
-  })
-
-  const isExpressCheckoutLoaded = Order2CheckoutContext.useStoreState(state => {
-    // Express Checkout is considered "loaded" if:
-    // 1. It's actually loaded (not null), OR
-    // 2. We're in post-payment state where Express Checkout should be hidden
-    const isActuallyLoaded = state.expressCheckoutPaymentMethods !== null
-    const activeStep = state.steps.find(
-      step => step.state === CheckoutStepState.ACTIVE,
-    )
-    const isInPostPaymentState =
-      activeStep?.name === CheckoutStepName.CONFIRMATION
-
-    return isActuallyLoaded || isInPostPaymentState
-  })
-
-  const isLoading = Order2CheckoutContext.useStoreState(
-    state => state.isLoading,
-  )
-  const setLoadingError = Order2CheckoutContext.useStoreActions(
-    actions => actions.setLoadingError,
-  )
-  const setLoadingComplete = Order2CheckoutContext.useStoreActions(
-    actions => actions.setLoadingComplete,
-  )
-
-  // Validate order and get into good initial checkout state on load
-  // - artwork version match
-  // - any resetting
-  useEffect(() => {
-    if (orderValidated || !orderData) {
-      return
-    }
-
-    try {
-      validateOrder(orderData)
-      setOrderValidated(true)
-    } catch (error) {
-      logger.error("Error validating order: ", error.message)
-      setLoadingError(error.message)
-    }
-  }, [orderData, orderValidated, setLoadingError])
-
-  useEffect(() => {
-    const timeout = setTimeout(() => {
-      setMinimumLoadingPassed(true)
-    }, MIN_LOADING_MS)
-    return () => clearTimeout(timeout)
-  }, [])
-
-  const isLoadingRef = useRef(isLoading)
-  useEffect(() => {
-    isLoadingRef.current = isLoading
-  }, [isLoading])
-
-  // biome-ignore lint/correctness/useExhaustiveDependencies: <explanation>
-  useEffect(() => {
-    const timeout = setTimeout(() => {
-      if (isLoadingRef.current) {
-        const error = new Error(
-          `Checkout loading state exceeded ${MAX_LOADING_MS}ms timeout: ${Object.entries(
-            {
```

Comment:
moved to `useLoadCheckout()` in the `Order2CheckoutApp` (where context is available)